download-center-upload/action.yml

```diff
@@ -25,6 +25,10 @@ inputs:
   artifact_file:
     description: Artifact(s) to upload. E.g. can contain a wildcard (*) or space separated string
     required: true
+  use_repo_name:
+    description: Variable which indicates whether the name of the repository should be included in the upload subpath.
+    default: 'true'
+    required: true
```

Comment:
Did you check that this works as expected? e.g. when `use_dc_repo` is not specified, does it work or crash?